fuse-products/src/main/java/software/tnb/product/csb/application/TomcatSpringBootApp.java

```diff
@@ -0,0 +1,194 @@
+package software.tnb.product.csb.application;
+
+import software.tnb.common.config.TestConfiguration;
+import software.tnb.product.application.App;
+import software.tnb.product.application.Phase;
+import software.tnb.product.csb.configuration.SpringBootConfiguration;
+import software.tnb.product.customizer.Customizer;
+import software.tnb.product.customizer.component.rest.RestCustomizer;
+import software.tnb.product.integration.builder.AbstractIntegrationBuilder;
+import software.tnb.product.log.FileLog;
+import software.tnb.product.log.Log;
+import software.tnb.product.log.stream.LogStream;
+import software.tnb.product.util.ZipUtils;
+import software.tnb.product.util.maven.BuildRequest;
+import software.tnb.product.util.maven.Maven;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.model.Dependency;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class TomcatSpringBootApp extends SpringBootApp {
+    private static final Logger LOG = LoggerFactory.getLogger(TomcatSpringBootApp.class);
+
+    private AbstractIntegrationBuilder<?> integrationBuilder;
+    private Path tomcatTmpDirectory;
+    private Path tomcatHome = null;
+    private Process tomcatProcess = null;
+    private static final String TOMCAT_PARENT_DIRECTORY = "tomcat";
+    private static final String TOMCAT_ARCHIVE_NAME = "tomcat-archive.zip";
+
+    @Override
+    public Log getLog() {
```

Comment:
set the `log` variable after the app is started, instead of overriding this method